### DISCUSSION TITLE
GitHub Badge for Verified Commit

### DISCUSSION THREAD

**Discussion by kiraware** [community/community#110795] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think GitHub Profile badge for verified commit is good for encouraging people to use PGP key signature on [signing commit](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and set it up on GitHub . See this article for setup verify commit https://daily-dev-tips.com/posts/how-to-verify-your-commits-on-github/.

**Comment by Blackie360** [comment]
Are  Github Profile badges that important ?

**Reply by kiraware** [reply]
Yes, it is important, because there are some people who hunt for GitHub Badge because it is attractive and beautiful. We can bait people into doing one of the best practices, namely verified commits, by simply giving a GitHub Badge to people who successfully push commits to GitHub with verified commits.
